scss/_footer.scss

```diff
@@ -0,0 +1,30 @@
+footer {
```

Comment:
Jeśli chcesz stosować `BEM` do nazewnictwa klas (widzę ten pattern na przykładzie `footer__privacy`) to nigdy nie nestuj innych bloków wewnątrz parent bloku. Czyli `row` nie ma racji bytu poniżej, zamiast tego powinien mieć własny plik ze stylami, albo powinien nazywać się `footer__row`